code/game/gamemodes/game_mode.dm

```diff
@@ -370,7 +370,7 @@
 //Reports player logouts//
 //////////////////////////
 /proc/display_roundstart_logout_report()
-	var/msg = "<span class='boldnotice'>Roundstart logout report\n\n</span>"
+	var/list/msg = list("<span class='boldnotice'>Roundstart logout report\n\n</span>")
 	for(var/i in GLOB.mob_living_list)
 		var/mob/living/L = i
 

```

Comment:
You could probably also filter here for ever having a mind (last_mind)

Reply:
Additionally, you can reduce loop size by iterating mob_living_list & player_list